debuglog-plugin/src/test/kotlin/com/bnorm/debug/log/IrPluginTest.kt

```diff
@@ -40,6 +47,9 @@ fun main() {
 
 @DebugLog
 fun greet(greeting: String = "Hello", name: String = "World"): String {
+    if (greeting != "Hello") {
```

Comment:
I thought it would be useful to see that the modified code also adds a println before an early return!

Reply:
I definitely like the idea of testing early return, but I would prefer to keep this particular source example really straight forward. Could early return (any even exception testing) be done as a different source file? I want the first thing people see in this file to be something really simple so maybe exit testing is a separate test class all together.